lib/class-wp-rest-menu-items-controller.php

```diff
@@ -284,7 +334,7 @@ protected function prepare_item_for_database( $request ) {
 				'menu-item-xfn'         => '',
 				'menu-item-status'      => 'publish',
 			);
-		}
+		}//end if
```

Comment:
Let's remove this, we don't follow this pattern in Core.

Reply:
The linter added these in for some reason. I can remove them, but I thought it was a new pattern in WP coding standards.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/071f64f3c82084544f6050f636b081eae460b344